performance/performance_tuning.rst

```diff
@@ -28,6 +28,155 @@ Hash based distribution is more suited to cases where users want to do real-time
 Once you choose the right distribution method and column, you can then proceed
 to the next step, which is tuning single node performance.
 
+.. _examining_plan:
+
+Examining Distributed Query Plan
+##################################
+
+Citus extends the SQL EXPLAIN command to provide information about simple distributed queries, meaning queries that do not contain re-partition jobs. The EXPLAIN output consists of two parts: the distributed query and the master query. These parts provide information to help tune Citus and the underlying PostgreSQL database.
```

Comment:
I think the [first paragraph from the PostgreSQL](https://citus-doc-preview-pr-94.herokuapp.com/performance/performance_tuning.html#postgresql-tuning) tuning section might be a good prelude to diving into the Explain plan. Maybe something to connect it like, "we can use EXPLAIN to visualize, and also tune for performance".